account_invoice_check_total/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Account Invoice Check Total
+===========================
+
+Add a Verification Total field on vendor bills.
+The user enters the taxes included invoice total as printed on the vendor bill, then enters the invoice lines and taxes. The system then checks the total computed by Odoo is the same as the verification total.
+
```

Comment:
Add runbot link in Usage section
Use template from maintainers repo (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)